pkg/cache/cache.go

```diff
@@ -766,6 +766,7 @@ func (c *Cache) LocalQueueUsage(qObj *kueue.LocalQueue) (*LocalQueueUsageStats,
 				if rf, ok := c.resourceFlavors[rgFlavor]; ok {
 					flavor.NodeLabels = rf.Spec.NodeLabels
 					flavor.NodeTaints = rf.Spec.NodeTaints
+					flavor.TopologyName = rf.Spec.TopologyName
```

Comment:
Please copy it conditionally, depending on TopologyAwareScheduling feature gate.

Reply:
Done